test/JDBC/expected/BABEL-ROLE-MEMBER-vu-verify.out

```diff
@@ -154,6 +154,27 @@ int
 0
 ~~END~~
 
+SELECT IS_MEMBER('BABEL_ROLE_MEMBER_vu_prepare_role1')
+GO
+~~START~~
+int
+0
+~~END~~
+
+SELECT IS_MEMBER('BABEL_ROLE_MEMBER_vu_prepare_role2')
+GO
+~~START~~
+int
+0
+~~END~~
+
+SELECT IS_MEMBER('BABEL_ROLE_MEMBER_vu_prepare_role3')
+GO
+~~START~~
+int
+0
+~~END~~
+
```

Comment:
Please add several tests here for `IS_ROLEMEMBER` as well, where we specify the second argument to be `db_owner` or `dbo`